client/express-checkout/utils/normalize.js

```diff
@@ -98,8 +98,34 @@ export const normalizeShippingAddress = ( shippingAddress ) => {
 		address_1: shippingAddress?.addressLine?.[ 0 ] ?? '',
 		address_2: shippingAddress?.addressLine?.[ 1 ] ?? '',
 		city: shippingAddress?.city ?? '',
-		state: shippingAddress?.region ?? '',
+		state: shippingAddress?.state ?? '',
 		country: shippingAddress?.country ?? '',
-		postcode: shippingAddress?.postalCode?.replace( ' ', '' ) ?? '',
+		postcode: shippingAddress?.postal_code ?? '',
 	};
 };
+
+/**
+ * Normalize Shipping Option used on PRBs to ECE format.
+ *
+ * @param {Object} option Shipping Option with PRB format.
+ *
+ * @return {Object} Shipping option with ECE format.
+ */
+export const normalizeShippinRate = ( option ) => {
+	return {
+		id: option.id,
+		amount: option.amount,
+		displayName: option.label,
+	};
+};
+
+/**
+ * Normalizes array of PRB Shipping Options objects to Shipping Rates with ECE format.
+ *
+ * @param {Array} options Array of Shipping Options with format to be used with PRB.
+ *
+ * @return {Array} Array of Shipping Rates formatted to be used with ECE.
+ */
+export const normalizeShippingRates = ( options ) => {
+	return options.map( normalizeShippinRate );
+};
```

Comment:
This isn't required; the shipping rates are already returned from the API correctly normalized.

Reply:
Removed in the merge.